src/wpanctl/tool-cmd-scan.c

```diff
@@ -56,6 +57,12 @@ print_scan_header(void)
 		printf("   ----+-------\n");
 
 		return;
+	} else if(sMleDiscoverScan) {
+		printf(
+				"   | NetworkName        | PAN ID | Ch | XPanID           | HWAddr           | RSSI\n");
+		printf(
+				"---+--------------------+--------+----+------------------+------------------+------\n");
+		return;
```

Comment:
Better to have a single exit from the end of the fn instead of returning midway through. Can you put an `} else {` , instead?